_analyzers/search-analyzers.md

```diff
@@ -67,14 +65,101 @@ PUT testindex
 ```
 {% include copy-curl.html %}
 
-## Specifying the default search analyzer for an index
+### Example: Using edge n-gram analyzer for indexing and standard analyzer for search
 
-If you want to analyze all query strings at search time with the same analyzer, you can specify the search analyzer in the `analysis.analyzer.default_search` setting. When providing the `analysis.analyzer.default_search`, you must also provide the `analysis.analyzer.default` parameter, which specifies the [index analyzer]({{site.url}}{{site.baseurl}}/analyzers/index-analyzers/) to be used at indexing time.
+The following configuration enables [autocomplete]({{site.url}}{{site.baseurl}}/search-plugins/searching-data/autocomplete/)-like behavior, where you can type the beginning of a word and still receive relevant matches:
 
-For example, the following request specifies the `simple` analyzer as the index analyzer and the `whitespace` analyzer as the search analyzer for the `testindex` index:
+```json
+PUT /articles
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "edge_ngram_analyzer": {
+          "tokenizer": "edge_ngram_tokenizer",
+          "filter": ["lowercase"]
+        }
+      },
+      "tokenizer": {
+        "edge_ngram_tokenizer": {
+          "type": "edge_ngram",
+          "min_gram": 2,
+          "max_gram": 10,
+          "token_chars": ["letter", "digit"]
+        }
+      }
+    }
+  },
+  "mappings": {
+    "properties": {
+      "title": {
+        "type": "text",
+        "analyzer": "edge_ngram_analyzer",
+        "search_analyzer": "standard"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The `edge_ngram_analyzer` is applied at index time, breaking input strings into partial prefixes (n-grams), which allows the index to store fragments like "se", "sea", "sear" and so on. 
+Use the following request to index a document:
 
 ```json
-PUT testindex
+PUT /articles/_doc/1
+{
+  "title": "Search Analyzer in Action"
+}
+```
+{% include copy-curl.html %}
+
+Use the following request to search for partial word `sear` in the `title` field:
+
+```json
+POST /articles/_search
+{
+  "query": {
+    "match": {
+      "title": "sear"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The response demonstrates that despite the query containing only partial word "sear", it matches the document "Search Analyzer in Action" because the n-gram tokens generated at index time include that prefix.
+This mirrors the [autocomplete functionality]({{site.url}}{{site.baseurl}}/search-plugins/searching-data/autocomplete/), where typing a prefix can retrieve full matches.
```

Comment:
```suggestion
This mirrors the [autocomplete functionality]({{site.url}}{{site.baseurl}}/search-plugins/searching-data/autocomplete/), in which typing a prefix can retrieve full matches:
```

Reply:
Please move line 132 to 131 so it's a 2-sentence line.

Reply:
@kolchfa-aws thats updated now